### ISSUE TITLE
Publish to docker hub (or other registry)

### ISSUE THREAD

**Issue by rodriciru** [StJudeWasHere/seonaut#25]
Do you have in mind to publish an image to docker hub, instead of clone git repo and make the image??
You can do in an automated way thought GitHub actions.

**Comment by NazgulCoder** [comment]
yes please, also get rid of mysql container and use MariaDB, it's more performant

**Comment by StJudeWasHere** [comment]
I'm closing this as the https://github.com/StJudeWasHere/seonaut/pull/36 pull request added the needed Github actions to publish the docker image to the Github registry. Docker images are now available in the packages section.
